src/webui/www/private/scripts/client.js

```diff
@@ -1594,4 +1663,25 @@ window.addEventListener("load", () => {
     window.qBittorrent.Cache.buildInfo.init();
     window.qBittorrent.Cache.preferences.init();
     window.qBittorrent.Cache.qbtVersion.init();
+
+    // switch to previously used tab
+    const previouslyUsedTab = LocalPreferences.get('selected_tab', 'transfers');
+    switch (previouslyUsedTab) {
+        case 'search':
+            if (window.qBittorrent.Client.isShowSearchEngine())
+                $('searchTabLink').click();
+            break;
+        case 'rss':
+            if (window.qBittorrent.Client.isShowRssReader())
+                $('rssTabLink').click();
+            break;
+        case 'log':
+            if (window.qBittorrent.Client.isShowLogViewer())
+                $('logTabLink').click();
+            break;
+        case 'transfers':
+        default:
```

Comment:
IMO the `default:` case should probably call `console.error()` to indicate it is an unexpected situation.
```js
default:
    console.error(`Unexpected `selected_tab` value: ${previouslyUsedTab}`);
    break;
```
I'll still approve the PR and you can change it if you want.

Reply:
I've added some logging to help with debugging.